test/e2e/test_auditlogs.go

```diff
@@ -89,11 +89,6 @@ func AuditLogsSourceWithTestImpl(t *testing.T) {
 			Kind:    "Service"}, targetName))
 	client.CreateAuditLogsOrFail(eventsAuditLogs)
 
-	// Create the Istio ServiceEntry.
-	istioServiceEntry := resources.IstioServiceEntry(
```

Comment:
if this is not used, then you can remove the resources.IstioServiceEntry code?

Reply:
Removed.